docs/assets/js/contact-form.js

```diff
@@ -0,0 +1,87 @@
+
+/* Contact form */
+$(function() {
+  var submitButton = $("#submit-button");
+  var form = $("#contact-form");
+  
+  selectPlanFromUrl();
+  
+  submitButton.on("click", submitForm);
+  
+  function selectPlanFromUrl() {
+    var params = window.location.search;
+    if (params === "?plan=enterprise") {
+      var plan = $("#enterprise");
+      plan.prop('checked', true);
+    } else {
+      var plan = $("#pro");
+      plan.prop('checked', true);
+    }
+  }
+
+  function submitForm(e) {
+    e.preventDefault();
+
+    if(validateForm()) {
+      var data = form.serialize(form.get(0), { hash: true });
+      submitToFormSpree(data)
+    }
+
+    function submitToFormSpree(data) {
+      submitButton.html("Sending...");
+      submitButton.prop("disabled", true);
+
+      var postParams = {
+        url: 'https://api.formbucket.com/f/buk_eldvLHDisTrizpsFPDfOm7fY',
```

Comment:
This should prob be read from the HTML form instead of duplicated in the JS code.

Reply:
We're fixing it right now ;)